assets/src/edit-story/components/header/buttons.js

```diff
@@ -17,10 +17,10 @@ import { useHistory, useStory } from '../../app';
 import { Outline, Primary, Undo, Redo } from '../button';
 
 const ButtonList = styled.nav`
-	background-color: ${ ( { theme } ) => theme.colors.bg.v3 };
 	display: flex;
 	justify-content: space-between;
-	padding: 1em;
+	padding-top: 20px;
```

Comment:
Really not a big fun of the vertical padding within the horizontal flexbox. What's the goal? To make these perfectly centered vertically? Let's then use `align-items`?

Reply:
Will revert this part for now.